test/powershell/scenarios.json

```diff
@@ -3,7 +3,7 @@
     "image": "mcr.microsoft.com/devcontainers/base:jammy",
     "features": {
       "powershell": {
-        "modules": "az.resources, az.storage",
+        "modules": "az.resources:7.1.0, az.storage",
```

Comment:
Can we create a new scenario for testing this change?

Reply:
created new test